ngraph/frontend/tensorflow/src/op/const.cpp

```diff
@@ -15,54 +15,56 @@ namespace op {
                                     const ov::element::Type>>;
 
 const ConstMap& TF_NGRAPH_CONST_MAP() {
     static const ConstMap the_map = {
-        {ov::element::f32, make_pair(MakeConstOp<float>, ov::element::f32)},
-        {ov::element::f64, make_pair(MakeConstOp<double>, ov::element::f64)},
-        {ov::element::i8, make_pair(MakeConstOp<int8_t>, ov::element::i8)},
-        {ov::element::i16, make_pair(MakeConstOp<int16_t>, ov::element::i16)},
+        {ov::element::f32, make_pair(make_const_op<float>, ov::element::f32)},
+        {ov::element::f64, make_pair(make_const_op<double>, ov::element::f64)},
+        {ov::element::i8, make_pair(make_const_op<int8_t>, ov::element::i8)},
+        {ov::element::i16, make_pair(make_const_op<int16_t>, ov::element::i16)},
 #if 0
-      {DataType::DT_QINT8, make_pair(MakeConstOp<qint8>, ov::element::i8)},
-      {DataType::DT_QUINT8, make_pair(MakeConstOp<quint8>, ov::element::u8)},
-      {DataType::DT_QUINT16, make_pair(MakeConstOp<quint16>, ov::element::u16)},
+      {DataType::DT_QINT8, make_pair(make_const_op<qint8>, ov::element::i8)},
+      {DataType::DT_QUINT8, make_pair(make_const_op<quint8>, ov::element::u8)},
+      {DataType::DT_QUINT16, make_pair(make_const_op<quint16>, ov::element::u16)},
 #endif
-        {ov::element::i32, make_pair(MakeConstOp<int32_t>, ov::element::i32)},
-        {ov::element::i64, make_pair(MakeConstOp<int64_t>, ov::element::i64)},
-        {ov::element::u8, make_pair(MakeConstOp<uint8_t>, ov::element::u8)},
-        {ov::element::u16, make_pair(MakeConstOp<uint16_t>, ov::element::u16)},
-        {ov::element::boolean, make_pair(MakeConstOp<bool, char>, ov::element::boolean)}
+        {ov::element::i32, make_pair(make_const_op<int32_t>, ov::element::i32)},
+        {ov::element::i64, make_pair(make_const_op<int64_t>, ov::element::i64)},
+        {ov::element::u8, make_pair(make_const_op<uint8_t>, ov::element::u8)},
+        {ov::element::u16, make_pair(make_const_op<uint16_t>, ov::element::u16)},
+        {ov::element::boolean, make_pair(make_const_op<bool, char>, ov::element::boolean)}
     };
     return the_map;
 }
 }  // namespace
 
-OutputVector TranslateConstOp(const NodeContext& node) {
+OutputVector translate_const_op(const NodeContext& node) {
     auto dt = node.get_attribute<ov::element::Type>("dtype");
-    Output<Node> ng_node;
+    Output<Node> res;
 
     // For some reason the following do not work (no specialization of
     // tensorflow::checkpoint::SavedTypeTraits...)
     // case DataType::DT_UINT32:
-    //   TF_RETURN_IF_ERROR(MakeConstOp<uint32>(op, element::u32,
+    //   TF_RETURN_IF_ERROR(make_const_op<uint32>(op, element::u32,
     //   &ng_node));
     //   break;
     // case DataType::DT_UINT64:
-    //   TF_RETURN_IF_ERROR(MakeConstOp<uint64>(op, element::u64,
+    //   TF_RETURN_IF_ERROR(make_const_op<uint64>(op, element::u64,
     //   &ng_node));
     //   break;
```

Comment:
propose to remove it and insert just `TODO: fix DT_UINT32 and DT_UINT64 support`

Reply:
I propose to fix it in the next PR

Reply:
fixed in https://github.com/openvinotoolkit/openvino/pull/8477